### PR TITLE
feat: integration_test with yaml config

### PR DESCRIPTION
**Pull request by Lvnszn**
**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Review by AlexStocks — APPROVED** [review]

File: integration_test/config/db_tbl/expected.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    - name: "sequence"
+      columns:
+        - name: "name"
+          type: "string"
+        - name: "value"
+          type: "string"
+data:
+  - name: "sequence"
+    value:
+      - ["1", "2"]
```
Comment (dongzl) [inline integration_test/config/db_tbl/expected.yaml:30]: add blank line at end of file.

File: integration_test/testcase/casetest.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      value: "1"
+exec_cases:
+  - sql: "INSERT INTO sequence(name,value,modified_at) VALUES(?,?,NOW())"
+    parameters: "1:string, 2:string"
+    sense:
+      - db
+      - tbl
+    expected:
+      type: "rowAffect"
+      value: 1
```
Comment (dongzl) [inline integration_test/testcase/casetest.yaml:45]: add blank line at end of file.

File: test/dataset.go [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+			}
+		}
+	}
+
+	for _, rows := range converts {
+		foundFlag := false
+		// for loop actual data
+		for _, actualRows := range actualMap {
+			for _, actualRow := range actualRows {
+				if actualRow[c[0]] == rows[0] {
```
Comment (dongzl) [inline test/dataset.go:170]: Maybe refactor a new method for data compare, reduce complexity.

**Review by dongzl — CHANGES_REQUESTED** [review]

File: cmd/main.go [hunk 4]
```diff
@@ -27,9 +27,7 @@ import (
 	_ "github.com/arana-db/arana/cmd/tools"
 )
 
-var (
-	Version = "0.1.0"
-)
+var Version = "0.1.0"
```
[the remark this answers is not in the document]
Comment (Lvnszn) [reply cmd/main.go:30]: gofumpt

**Review by jjeffcaii — APPROVED** [review]

File: integration_test/config/tbl/expected.yaml [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    - name: "order"
+      columns:
+        - name: "name"
+          type: "string"
+        - name: "value"
+          type: "string"
+data:
+  - name: "order"
+    value:
+      - ["test", "test1"]
```
[the remark this answers is not in the document]
Comment (Lvnszn) [reply integration_test/config/tbl/expected.yaml]: addressed

File: test/dataset.go (hunk 3 above)
Comment (Lvnszn) [reply test/dataset.go]: addressed

**Review by dongzl — APPROVED** [review]

**Pull request merged**